pkg/cloudprovider/aws/suite_test.go

```diff
@@ -419,8 +419,8 @@ var _ = Describe("Allocation", func() {
 			})
 			It("should support a capacity type label", func() {
 				for _, value := range []string{v1alpha1.CapacityTypeOnDemand, v1alpha1.CapacityTypeSpot} {
-					provisioner.Spec.Labels = map[string]string{v1alpha1.CapacityTypeLabel: value}
-					Expect(provisioner.Validate(ctx)).ToNot(Succeed())
+					provisioner.Spec.Labels = map[string]string{v1alpha5.LabelCapacityType: value}
+					Expect(provisioner.Validate(ctx)).To(Succeed())
```

Comment:
Why was this false before?

Reply:
Honestly, i was hoping you would know since you had done this XD https://github.com/awslabs/karpenter/commit/67d637a3c3a2db5eb6c23801f3c12e79b8948f08#diff-d9acf1c0f1f6b83a9cecc0e967d90f0a2fd5bc6e7886d0607c05b533ad905828R415

Which was a re-factor of code that you also wrote: https://github.com/awslabs/karpenter/commit/69894662390c0fc3c4c7f07b5896055ebcfe9e43#diff-d9acf1c0f1f6b83a9cecc0e967d90f0a2fd5bc6e7886d0607c05b533ad905828R452

If I had to guess, you made a copy/paste mistake and so there was a legitimate bug in the code that was being missed because of this test. 

It didn't fail until my changes this morning so I expect is has something to do with promoting capacity type to be a well known label. But when I un-do that promotion other tests fail as well so I can't easily pinpoint what I did that fixed this.

Reply:
Re-reading this I feel like I just threw this back in your face, not my intention - sorry! My point was to say: I couldn't figure it out, and didn't get a chance to ping you. I had figured out it was a test you made but didn't understand the test itself. My point was: I think you made an honest mistake in your original test and I happened to fix that bug in my PR. If you think that your original code was correct, would love help figuring out what I'm getting lost on here.